service/src/main/java/org/mskcc/smile/service/impl/SampleServiceImpl.java

```diff
@@ -183,28 +184,8 @@ public SmileSample fetchAndLoadPatientDetails(SmileSample sample) throws Excepti
                     + "- confirm whether data construction and parsing is being handled correctly");
         }
 
-        // scenario that requires a patient swap and updating the sample-to-patient relationship
-        // in the database and removing the former sample-to-patient relationship
-        if (patientByLatestCmoId == null) {
-            SmilePatient newPatient = new SmilePatient(sampleMetadata.getCmoPatientId(), "cmoId");
-            //if this sample is a clinical sample, we would also need to add dmpId
-            if (sample.getSampleCategory().equals("clinical")) {
-                Matcher matcher = DMP_PATIENT_ID.matcher(sampleMetadata.getPrimaryId());
-                if (matcher.find()) {
-                    newPatient.addPatientAlias(new PatientAlias(matcher.group(), "dmpId"));
-                }
-            }
-            patientService.savePatientMetadata(newPatient);
-            sample.setPatient(newPatient);
-            // remove sample-to-patient relationship from former patient node
-            sampleRepository.removeSamplePatientRelationship(sample.getSmileSampleId(),
-                    patient.getSmilePatientId());
-            return sample;
-        }
-
-        // scenario where we are checking for an update to the existing patient, which is the same
-        // that already existing and is linked to the sample in the database but may contain updates
-        // (i.e., a new patient alias)
+        // scenario where we are checking for an update to the existing patient and is linked to the
+        // sample in the database but may contain updates (i.e., a new patient alias)
```

Comment:
code handling updates to an existing patient takes precedence over code handling a patient swap scenario